sw/device/lib/testing/test_framework/ottf_main.h

```diff
@@ -8,7 +8,6 @@
 #include <stdbool.h>
 
 #include "external/freertos/include/FreeRTOS.h"
```

Comment:
Do we need to keep this include?

Reply:
Unfortunately, we need to keep this include because the `TaskFunction_t` type is defined in it, which is in the declaration of the `ottf_task_create` function.